src/app/modules/auth/components/register-form/register-form.component.html

```diff
@@ -0,0 +1,52 @@
+<form action="">
+  <mat-form-field>
+    <input matInput placeholder="First Name" required name="firstName" #firstName
+      [(ngModel)]="userRegisterRequest.firstName" [ngModelOptions]="{standalone: true}"
+      (blur)="nameChanged('firstName',$event)">
```

Comment:
Is it necessary to validate the field on Blur? Validating on submit button click will also work fine. What do you think?
So directly validating from the form model (than from userRegisterFromError will work right?)